package.json

```diff
@@ -14,6 +14,7 @@
     "google-maps-react": "^2.0.2",
```

Comment:
Please be really careful with running a command like $ npm update, since this usually leads to a lot of merge conflicts in package-lock.json (which it did this time).